src/main.ts

```diff
@@ -66,53 +63,44 @@ class DrawingApp {
       img.height * ratio
     );
   }
-  private redraw() {
+
+  private redraw(): void {
     const clickX = this.clickX;
-    const context = this.context;
-    //context.drawImage(this.img, 0, 0);
-    const clickDrag = this.clickDrag;
     const clickY = this.clickY;
+    const context = this.context;
+
     for (let i = 0; i < clickX.length; ++i) {
       context.beginPath();
-      if (clickDrag[i] && i) {
-        context.moveTo(clickX[i - 1], clickY[i - 1]);
-      } else {
-        context.moveTo(clickX[i] - 1, clickY[i]);
-      }
-
-      context.lineTo(clickX[i], clickY[i]);
-      context.strokeStyle = '#ff0000';
+      context.arc(clickX[i], clickY[i], 10, 0, 2 * Math.PI);
```

Comment:
```suggestion
      context.arc(clickX[i], clickY[i], 2.5, 0, 2 * Math.PI);
```

would be nice to make the circles more similar to the size of colonies in the test image